app/models/certificate.rb

```diff
@@ -82,6 +86,10 @@ def for_surveys(surveys)
     end
   end
 
+  def init
+    self.attained_level ||= 'none'
+  end
+
```

Comment:
just checked, adding a column default, sets the value to the default in new models. Can't quite find it in the `ActiveRecord` code though because that is a maze of delegation. Something to do with `AttributeSet#_default_attributes` though